Formula/packetq.rb

```diff
@@ -0,0 +1,26 @@
+class Packetq < Formula
+  desc "SQL-frontend to PCAP-files"
+  homepage "https://www.dns-oarc.net/tools/packetq"
+  url "https://github.com/DNS-OARC/PacketQ/archive/v1.3.1.tar.gz"
```

Comment:
@tomquas In this instance, it is plain that the link given in the text of the release is the correct source tarball. A clear indicator of this is that it can be built from source without having to first run the `autogen.sh` script.

Also, as I alluded to before, but didn't actually directly state, the tarballs under the `Downloads` header that you a referring to are automatically created from the repository by GitHub when the release is tagged.